src/tests/queries.rs

```diff
@@ -298,6 +304,7 @@ async fn test_get_items_by_currency() {
 
 #[tokio::test]
 async fn test_get_advanced_items() {
+    Lazy::force(&INIT);
```

Comment:
Why are you doing this ?

There is no code that needs to know that we are in a Test.

Also if you delete this line, you can delete the 2 dependencies that you added.

Reply:
It is necessary in order to display the log::error! messages in case of error. 
I re factorized it using `lazy_static` to avoid using once_cell, but i still need to initialize it at the start of the test using

```rust
lazy_static::initialize(&INIT);
```

Here is the command i am using for running the tests:
```
RUST_LOG=error cargo test --package vinted-rs --lib -- tests::queries::test_get_advanced_items --exact --show-output 
```

Reply:
As u told me, I decided to avoid using more crates for this and, since its just for one test, I added the init line as follows:
```rust
    let _ = env_logger::builder().is_test(true).init();
```
to the start of the function.
So weird cargo hasn't added something like `[#setup]` for running code before every test.